shopinvader/views/shopinvader_product_view.xml

```diff
@@ -47,9 +47,9 @@
         <field name="model">shopinvader.product</field>
         <field name="arch" type="xml">
             <tree>
-                <field name="backend_id" />
-                <field name="lang_id" />
                 <field name="name" />
+                <field name="backend_id" />
+                <field name="lang_id" optional="show" />
```

Comment:
lang should not be optional IMO

Reply:
It's optional but it's shown by default. Just gives the user the possibility to hide it.
I guess a better approach would be to do something like this:
https://github.com/odoo/odoo/blob/a5077b300c5cb5eae59e0bb67a5d5429b846f642/odoo/addons/base/views/res_partner_views.xml#L265-L266

But it's an overkill IMO, no?

Reply:
overkill, just keep it :)